trail/query/rest_views.py

```diff
@@ -1,24 +1,45 @@
 import django
 from django.core.exceptions import FieldError
 
-import numpy as np
-
 from rest_framework import mixins, status
-from rest_framework import exceptions as rf_error
 from rest_framework.views import APIView
 from rest_framework.response import Response
 
 from upload.models import Metadata, Wcs
 from query.serializers import MetadataSerializer, WcsSerializer
 
-from drf_yasg.utils import swagger_auto_schema
-
 
 class GenericTableView(mixins.ListModelMixin, APIView):
+    """
+     GenericTableView that serves as an interface to provide one unify rest api query
+     to different tables
+    """
+
```

Comment:
Should probably describe it requires specifying `queryset` and `serializer_class` class attributes for the table view to work.